src/contexts/W3iContext/hooks/notifyHooks.ts

```diff
@@ -44,12 +46,18 @@ export const useNotifyState = (w3iProxy: Web3InboxProxy, proxyReady: boolean) =>
     })
   }, [notifyClient, userPubkey, proxyReady])
 
+  // it takes time for handshake (watch subscriptions) to complete
+  // load in progress state using interval until it is
   useEffect(() => {
+    if(watchSubscriptionsComplete) {
+      return noop;
+    }
+
     // Account for sync init
-    const timeoutId = setTimeout(() => refreshNotifyState(), 100)
+    const timeoutId = setInterval(() => refreshNotifyState(), 500)
```

Comment:
Depending on the cost of the fetching notify state on the server, I think this could be even `1000` or `2000`,  wdyt?

Reply:
This does not fetch from notify server, just locally